android-project/app/build.gradle

```
@@ -65,4 +65,5 @@ android {
 dependencies {
 	implementation fileTree(include: ['*.jar'], dir: 'libs')
 	implementation 'com.android.support.constraint:constraint-layout:2.0.4'
+	implementation 'org.jetbrains:annotations:15.0'

```

Should this be a compileOnly dependency? Doesn't look like it's needed at runtime.

---

Don't know much about Java, it was added automatically by Android Studio (Jetbrain) when i added the annotation suggestion. If I had know that it was the automatic fixes that you guys thought where controversial I would have just left them out :joy:

---

😂 surprised jetbrain doesn't use their own recommended config

https://github.com/JetBrains/java-annotations

```suggestion
	compileOnly 'org.jetbrains:annotations:23.0.0'
```

---

It did also initially add them in there long form and without importing the namespace, had to google a bit to make it compile :roll_eyes: